### PR TITLE
chore: add meta tags for OG and Twitter

### LINKED ISSUE #149: Create og:image meta image design
http://studio.asyncapi.com/

**Examples:**

### PR DESCRIPTION
**Pull request by mcturco**
**Description**

- Added meta title
- Added meta tags for OG / Facebook
- Added meta tags for Twitter

**Related issue(s)**
Resolves #149

### CODE REVIEW

**Comment by mcturco** [comment]
@magicmatatjahu just checked that the open graph image is showing here: https://www.opengraph.xyz/url/https%3A%2F%2Fdeploy-preview-170--modest-rosalind-098b67.netlify.app%2F/
It works with relative path in the content, but if there is a better way to do this to output the whole url let me know. I tried using `%PUBLIC_URL%` in the meta but it wasn't returning anything

**Comment by magicmatatjahu** [comment]
@mcturco No problem, there can be relative path. Thanks very much!

**Review by magicmatatjahu — APPROVED** [review]

**Comment by magicmatatjahu** [comment]
Before merge. Why do you wanna use absolute paths, for SEO reason? @mcturco

File: public/index.html [hunk 1]
```diff
@@ -5,10 +5,26 @@
...
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="/">
+    <meta property="og:title" content="AsyncAPI Studio">
+    <meta property="og:description" content="Studio for AsyncAPI specification, where you can validate, view preview documentation, and generate templates from AsyncAPI document.">
+    <meta property="og:image" content="/img/meta-studio-og-image.jpg">
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summare_large_image">
+    <meta property="twitter:url" content="/">
```
Comment (magicmatatjahu) [inline public/index.html:23]: I think that url should point to the twitter account like in conference page: 
```suggestion
    <meta property="twitter:url" content="https://twitter.com/asyncapispec">
```

and we should also add domain property:

```
    <meta property="twitter:domain" content="https://studio.asyncapi.com">
```

**Comment by mcturco** [comment]
I believe it is better practice to have absolute paths in the meta tags... however, it seems to be passing the Open Graph tests.

File: public/index.html (hunk 1 above)
Comment (mcturco) [reply public/index.html:23]: So I actually think the `twitter:url` property is wrong. I think it is actually supposed to be `twitter:site` as shown here: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup

And I just noticed that I have a typo on line 22 - `summare_large_image` is supposed to be `summary_large_image` 😅 

Also, I don't think we need `twitter:domain` as I don't see it in the twitter docs. Where did you find that property?

Comment (magicmatatjahu) [reply public/index.html:23]: Ok, so it should be site :) I found domain here https://www.opengraph.xyz/url/https%3A%2F%2Fdeploy-preview-170--modest-rosalind-098b67.netlify.app%2F/ at the bottom of page:

```
<!-- Twitter Meta Tags -->
<meta name="twitter:card" content="summary_large_image">
<meta property="twitter:domain" content="deploy-preview-170--modest-rosalind-098b67.netlify.app">
...
```

Comment (mcturco) [reply public/index.html:23]: hmmm okay. I will make those changes!

File: public/index.html [hunk 2]
```diff
@@ -5,10 +5,27 @@
...
     <meta name="theme-color" content="#000000" />
+    <meta name="title" content="AsyncAPI Studio">
     <meta
       name="description"
       content="Studio for AsyncAPI specification, where you can validate, view preview documentation, and generate templates from AsyncAPI document."
     />
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="/">
```
Comment (magicmatatjahu) [inline public/index.html:16]: Ok last thing @mcturco 😅 You can push changes via github :)
```suggestion
    <meta property="og:url" content="https://studio.asyncapi.com/">
```

**Review by magicmatatjahu — APPROVED** [review]

**Comment by magicmatatjahu** [comment]
@mcturco Thanks!

**Pull request merged**